src/gp_activetable.c

```diff
@@ -236,7 +252,7 @@ report_altered_reloid(Oid reloid)
 	if (IsRoleMirror() || IS_QUERY_DISPATCHER()) return;
 
 	LWLockAcquire(diskquota_locks.altered_reloid_cache_lock, LW_EXCLUSIVE);
-	hash_search(altered_reloid_cache, &reloid, HASH_ENTER, NULL);
+	ALTERED_RELOID_CACHE_ENTER(&reloid, NULL);
```

Comment:
Could you avoid macros here?

Reply:
I think it's better to leave it uniform with the rest of the inserts in the hash table.

Reply:
However, I think that such macros are unintuitive because they hide the variables used internally.

Reply:
how about such
```suggestion
	shm_hash_enter(altered_reloid_cache, &reloid, NULL, diskquota_max_active_tables, ALTERED_RELOID_CACHE_WARNING, &altered_reloid_cache_last_overflow_report, diskquota_max_active_tables);
```

Reply:
the task of the macro is to hide the parameters that are always the same. it may make sense to remove the macro for maps in which the insertion occurs only in one place. but it seems to me to be a complication.